WordPress/src/main/java/org/wordpress/android/ui/stats/StatsBarGraph.java

```diff
@@ -253,6 +263,11 @@ public void run() {
         }, 500);
     }
 */
+
+    public void setWeekendDays(boolean[] days) {
+        mWeekendDays = days;
```

Comment:
At first I was worried we didn't `clone()` here, but the implementation doesn't require it :+1:

Arrays are technically `Object`'s in Java so it's something to keep an eye on.